static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -22,73 +18,63 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   const organization = useOrganization();
   const project = useProjectFromSlug({organization, projectSlug: event['project.name']});
   const issueId = event['issue.id'];
-  // We are view issue X and loading data for issue Y. This means that this call will not be cached,
-  // thus, it is a little bit slow to render the component.
-  // XXX: Create new endpoint that only fetches the metadata for the group
-  const {data: groupData, isLoading: isLoadingGroupData} = useApiQuery<Group>(
-    [
-      `/organizations/${organization.slug}/issues/${issueId}/`,
-      {query: getGroupDetailsQueryData({})},
-    ],
-    {
-      staleTime: 30000,
-      cacheTime: 30000,
-      retry: false,
-    }
-  );
+  const {title, subtitle} = getTitleSubtitle(event);
   const avatarSize = parseInt(space(4), 10);
 
   // If any of data fails to load, we don't want to render the component
   // Only "One other issue appears in the same trace. View Full Trace (X issues)" would show up
   return (
     <Fragment>
-      {isLoadingGroupData ? (
-        <LoadingIndicator mini />
-      ) : (
-        groupData && (
```

Comment:
I'm removing these 4 lines. Hiding whitespaces makes it more obvious.
<img width="201" alt="image" src="https://github.com/getsentry/sentry/assets/44410/e8a646f9-ed69-4a9b-837a-4001c1b05de9">